c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -204,6 +204,8 @@ struct Block {
   c10::DeviceIndex device; // gpu
   cudaStream_t stream; // allocation stream
   stream_set stream_uses; // streams on which the block was used
+  stream_set cudagraph_stream_uses; // streams on which the block was
```

Comment:
This is very rarely going to be set on blocks, and only temporarily. Better to have as `unordered_map` instead ?